ast/Serializer.go

```diff
@@ -1535,11 +1536,16 @@ func (meta *ConstantMeta) ReadMetaFrom(reader io.Reader) error {
 		return err
 	}
 	byteArr := make([]byte, length)
-	_, err = reader.Read(byteArr)
+
+	readCount, err = reader.Read(byteArr)
```

Comment:
I am very sorry, I committed the code wrongly. I should be
readCount, err := reader.Reader(byteArr)

Forgot to puth the collon there. Coud you change it accordingly and push again.